### ISSUE TITLE
Add Support for Helix Editor

### ISSUE THREAD

**Issue by SunP04** [elbywan/crystalline#93]
As of right now, when using the crystalline LSP in helix, the lsp just closes, without giving any output or notice.

Since I don't know the cause or whose problem it is in the current moment, I created an issue on both platforms (Crystalline and Helix) for both removing it as a supported lsp and/or solving the giving issue of it.

Helix issue can be found here: https://github.com/helix-editor/helix/issues/11301

**Comment by uncomfyhalomacro** [comment]
@SunP04 have you tested this? it works as long as crystalline knows where the crystal libs are i.e. outside of distrobox, crystalline will not know where the crystal libs are since crystal is installed within a podman container and not the host.

**Comment by uncomfyhalomacro** [comment]
It does not work on the following editors though:
- kakoune with kakoune-lsp
- neovim (?)

**Comment by SunP04** [comment]
I can't test it right now, maybe in some hours. But I used a win10 and I'm pretty sure everything was in path correctly, as I've used `hx --health` to confirm.

Might be a windows only issue, but that doesn't make much sense to me.

**Comment by uncomfyhalomacro** [comment]
> I can't test it right now, maybe in some hours. But I used a win10 and I'm pretty sure everything was in path correctly, as I've used `hx --health` to confirm.
> 
> Might be a windows only issue, but that doesn't make much sense to me.

Yep. although, it does not hinder me much on crystal dev. i am just sad i can't use my fav editor and resort to vscode.

>  it works as long as crystalline knows where the crystal libs are

this is confirmed on my side but seeing this

![image](https://github.com/user-attachments/assets/ebd2d8cd-fd6f-4638-82e7-6cf735be58f6)

made me think that this is crystalline not supporting windows

**Comment by igbanam** [comment]
@uncomfyhalomacro I've been using Crystalline in Vim since I knew about it. The setup is

- Vim 9
- coc.nvim
- Crystalline binary (`brew install ...`)

```
  "languageserver": {
    "crystal": {
      "command": "crystalline",
      "args": [
        "==stdio"
      ],
      "filetypes": [ "crystal" ],
      "rootPatterns": [ "shard.yml" ]
    },
```
…inside `~/.vim/coc-settings.json`

Something similar should work in Neovim for whatever LSP client there
